AAT_gae/src/main/java/com/aat/restlet/AATApplication.java

```diff
@@ -1,17 +1,36 @@
 package com.aat.restlet;
 
+import org.restlet.engine.Engine;
+import org.restlet.engine.converter.ConverterHelper;
+import org.restlet.ext.jackson.JacksonConverter;
 import org.restlet.Application;
 import org.restlet.Restlet;
 import org.restlet.routing.Router;
 
 import com.aat.restlet.GroupsResource;
 import com.aat.utils.Constants;
 
+import java.util.List;
+
+
 
 public class AATApplication extends Application {
 	
 	@Override
 	public Restlet createInboundRoot() {
+		// Change the Jackson converter to use the ObjectifyJacksonModule mapper
+		List<ConverterHelper> converters = Engine.getInstance().getRegisteredConverters();
+		JacksonConverter jc = null;
+		for (ConverterHelper converter: converters) {
+			if (converter instanceof JacksonConverter) {
+				jc = (JacksonConverter) converter;
+			}
+		}
+		if (jc != null) {
```

Comment:
To be honest, I pulled this part from a tutorial on how to change the Jackson converter to use a custom mapper. This part is just to make sure that the keys in the Objectify objects don't end up messing up the Jackson serialization with their self-references.